homeassistant/data_entry_flow.py

```diff
@@ -122,11 +124,12 @@ class FlowHandler:
     flow_id = None
     hass = None
     handler = None
-    source = None
     cur_step = None
+    context = None
 
     # Set by _async_create_flow callback
     init_step = 'init'
+    source = None
```

Comment:
Remove this since it's in context now?

Reply:
I was just lazy to change... will do it.